### PR TITLE
webpack config changed for url and file loaders

### PR DESCRIPTION
**Pull request by PrudhviRaj5**

### CODE REVIEW

**Comment by PrudhviRaj5** [comment]
Hi @KleoPetroff,
Are you planning to merge this or you want to keep this as a separate branch? I can add history fallback and redux next. Even if the Redux is added, people can still use React state management and it's always optional. Please tell me what I should do next?

**Comment by KleoPetroff** [comment]
I'm planning to merge it after I review it since it contains the file loader fix. It was better to create another PR just for the dependency updates, as I have some considerations about updating eslint, which may delay the merge.

You can go for the history fallback next. As for Redux, I don't plan to add it to `master`. You can create a separate branch for the implementation.

File: package.json [hunk 1]
```diff
@@ -18,57 +18,61 @@
...
     "babel-preset-stage-2": "6.24.1",
-    "copy-webpack-plugin": "4.1.1",
+    "copy-webpack-plugin": "4.2.3",
     "css-loader": "0.28.7",
     "enzyme": "3.2.0",
     "enzyme-adapter-react-16": "1.1.0",
-    "eslint": "3.19.0",
-    "eslint-config-airbnb": "15.1.0",
-    "eslint-loader": "1.8.0",
+    "eslint": "4.12.1",
```
Comment (KleoPetroff) [inline package.json]: does `4.21.1` work with the `eslint-loader`? Have you tested it

Comment (PrudhviRaj5) [reply package.json]: Yes! I have tested everything (with webpack dev & prod) and have pushed the code. You can pull the branch and check it once if you like.

Comment (KleoPetroff) [reply package.json]: Nice!

**Comment by KleoPetroff** [comment]
Thank you again for the wonderful work you did. :)

**Pull request merged**